cNVMe/Identify.h

```diff
@@ -0,0 +1,201 @@
+/*
+###########################################################################################
+// cNVMe - An Open Source NVMe Device Simulation - MIT License
+// Copyright 2017 - Intel Corporation
+
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation the
+// rights to use, copy, modify, merge, publish, distribute, sublicense, and/or
+// sell copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+// PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT
+// OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+// OTHER DEALINGS IN THE SOFTWARE.
+############################################################################################
+Identify.h - A header file for the NVMe Identify Commands
+*/
+
+#include "Types.h"
+
+#pragma once
+
+namespace cnvme
+{
+	namespace identify
+	{
+		namespace structures
```

Comment:
Is it worth doing another namespace here?

Reply:
I think this is a sufficient number of namespaces (and its plenty descriptive as is).

Reply:
I think this is a sufficient number of namespaces (and its plenty descriptive as is).